.github/workflows/python_build.yaml

```diff
@@ -0,0 +1,85 @@
+name: Build Wheels
+
+on:
+  workflow_dispatch:
+
+jobs:
+  python-ci:
+    if: github.event_name == 'workflow_dispatch' && startsWith(github.ref, 'refs/tags')
+    uses: ./.github/workflows/python_ci.yaml
+
+  build_matrix:
+    strategy:
+      matrix:
+        python_version: [3.9, 3.11]
```

Comment:
Another thought I had here, any downside to doing this for all the python versions we support right now?